_posts/2024-08-02-gsoc2024_talagayev.md

```diff
@@ -0,0 +1,193 @@
+---
+layout: post
+title: GSoC 2024 - 2D visualization for small molecules
+---
+
+**Contributor**: Valerij Talagayev ([@talagayev](https://github.com/talagayev))
+
+**Mentors**: Cédric Bouysset ([@cbouy](https://github.com/cbouy)), Richard Gowers ([@richardjgowers](https://github.com/richardjgowers)) and Yuxuan Zhuang ([@yuxuanzhuang](https://github.com/yuxuanzhuang))
+
+**Organization**: [MDAnalysis](https://www.mdanalysis.org/)
+
+**Release**: [GSoC 2024: 2D visualization for small molecules Release](https://github.com/talagayev/MDonatello/releases/tag/0.0.1)
+
+During the Google Summer of Code 2024 I was working on the 2D Visualization of small molecules, which is an important project, that would allow people to easily visualize the molecule that they have in their file
+through the selection of the molecule as an `AtomGroup` in `MDAnalysis`.
+
+## Goals of the project
+
+The goal of the project was to create a code, that would allow the user to select an `AtomGroup` with the help of `MDAnalysis`, which would be used with an RDKit Converter to convert the molecule into an rdkit mol object, which would be used for the visualization.
```

Comment:
```suggestion
The goal of the project was to create a code, that would allow the user to select an `AtomGroup` with the help of `MDAnalysis`, which would be used with an [RDKit](https://www.rdkit.org/) Converter to convert the molecule into an rdkit mol object, which would be used for the visualization.
```

Reply:
Perhaps link to RDKit somewhere as well?